apps/admin/oauth2-proxy/oauth2-proxy.yaml

```diff
@@ -19,6 +19,7 @@ spec:
   values:
     extraArgs:
       cookie-domain: ".demo.platform.hmcts.net"
+      session-cookie-minimal: true
```

Comment:
see
https://github.com/oauth2-proxy/oauth2-proxy/issues/644
and
https://github.com/oauth2-proxy/oauth2-proxy/blob/965fab422d6cd30411a6ee85a34532b276284289/pkg/apis/options/options.go#L137